serverless/src/index.ts

```diff
@@ -89,7 +89,10 @@ export const handler = async (event: InputEvent, _: any) => {
       };
     }
 
-    const lambdas = findLambdas(resources, event.templateParameterValues);
+    // find lambdas then filter out the lamba's that are flagged to be excluded from instrumentation.
+    const lambdas = findLambdas(resources, event.templateParameterValues).filter(
+      (lambda) => !config.exclude?.includes(lambda.key),
```

Comment:
Not sure if we should have a `?` here, what does the `serverless-plugin-datadog` do for this filtering?

Reply:
I am using an optional chain here so that if the exclude parameter is undefined it will just ignore filter and instrument all of the defined handlers. This _could_ be removed now that we are setting a default value of an empty array. 

For curiosity's sake I did look at how the [serverless plugin](https://github.com/DataDog/serverless-plugin-datadog/blob/c6f1c6177bfdca9dced0aae83fc491880d7e1a99/src/layer.ts#L110
) implements the handler filter logic. It looks like its explicitly doing undefined checks which I feel are unneeded when using optional chains. 

I don't think keeping this in will have negative effects. 

But I welcome any feedback.